includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -403,6 +403,19 @@ private function adjust_and_replace_node( Element $node ) {
 		if ( $this->args['native_img_used'] || ( $node->parentNode instanceof Element && Tag::PICTURE === $node->parentNode->tagName ) ) {
 			$attributes = $this->maybe_add_lightbox_attributes( [], $node ); // @todo AMP doesn't support lightbox on <img> yet.
 
+			/*
+			 * Mark lightbox as px-verified attribute until it's supported by AMP spec.
+			 * @see <https://github.com/ampproject/amp-wp/issues/7152#issuecomment-1157933188>
+			 * @todo Remove this once lightbox is added in `ightboxable-elements` for native img tag in AMP spec.
+			 */
+			if ( isset( $attributes['data-amp-lightbox'] ) ) {
+				$node->setAttribute( Attribute::LIGHTBOX, 'true' );
+				$node_attr = $node->getAttributeNode( Attribute::LIGHTBOX );
```

Comment:
```suggestion
				$node_attr = $node->getAttributeNode( Attribute::LIGHTBOX );
				if ( ! $node_attr instanceof DOMAttr ) {
					$node_attr = $dom->createAttribute( Attribute::LIGHTBOX );
					$node->setAttributeNode( $node_attr );
				}
```

Then no `if` is needed below.

Reply:
`$dom->createAttribute` will be `$this->dom->createAttribute` 😄 . So created a seperate commit.